src/ralph/admin/templates/admin/includes/javascripts.html

```diff
@@ -42,7 +42,7 @@
                   $(item).val($original_selects.eq(index).val());
                 });
                 $clone.appendTo($form);
-                $form.submit();
+                $form[0].submit();
```

Comment:
`$form.submit` is `jquery` version of `submit`. I'm satisfied with regular `submit` which is accesible on node.